packages/at_onboarding_cli/example/README.md

```diff
@@ -0,0 +1,29 @@
+List of steps to run the examples for checking apkam enrollment
```

Comment:
I have requested @srieteja to go over this readme and test the flow. Replace normal args with ArgParser and usage